src/github.com/t3chguy/matrix-static/matrix-static.go

```diff
@@ -61,6 +69,41 @@ func main() {
 		pprof.Register(router, nil)
 	}
 
+	avatarRouter := router.Group(*publicServePrefix)
+	avatarRouter.Use(gin.Recovery())
+
+	generatedAvatarCache := persistence.NewInMemoryStore(0)
```

Comment:
I'm not seeing any cache rules here. I hope we don't eventually just stack up images of every single unicode character...